src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -26,7 +39,6 @@ md-progress-linear {
   }
 }
 
-.kd-deploy-input-row {
-  margin-bottom: $baseline-grid;
-  margin-top: $baseline-grid;
+.progress-or-message-container {
```

Comment:
Hmmm... I dont know why we neeed this. For me the progress bar is not jittering. What browser do you use?